### DISCUSSION TITLE
Best Practice for Text overlay on Image contained in a box?

### DISCUSSION THREAD

**Discussion by tae-lim** [chakra-ui/chakra-ui#4219] [General]
Hi, I'm new to ChakraUI and am working on a new project. 

Let's say we have a Link Overlay as a card box. I have an image that I want to fit within the full width and height of the box, but want text on top of the image. Is there an easy way to do this?

**Comment by anubra266** [comment]
GIve the Box relative position, give the text an `absolute` position, and the image comes on top of the Box and Image .

```html
<Box pos="relative" boxSize="400px">
    <Image src="https://bit.ly/2Z4KKcF" boxSize="full" />
    <Text pos="absolute" top="50%" left="50%" color="red" transform="translate(-50%,-50%)">
      This text is centered on the image
    </Text>
</Box>
```
